R/build_lm.R

```diff
@@ -257,12 +257,12 @@ build_lm.fast <- function(df,
 
           c_cols <- c(c_cols, absolute_time_col, wday_col, day_col, yday_col, month_col, year_col)
           df[[absolute_time_col]] <- as.numeric(df[[col]])
-          # turn it into character since if it is factor, the name of term is broken
-          df[[wday_col]] <- as.character(lubridate::wday(df[[col]], label=TRUE))
+          # turn it into unordered factor since if it is ordered factor, the name of term is broken
+          df[[wday_col]] <- factor(lubridate::wday(df[[col]], label=TRUE), ordered=FALSE)
```

Comment:
Why do you specify ordered=FALSE explicitly here?

Reply:
added comment.

Reply:
I'm sorry but I can't understand the reason why it needs to be unordered factor by this comment.